src/en/news/blog/2023/v17-2-6-quincy-released/index.md

```diff
@@ -0,0 +1,1052 @@
+---
+title: "v17.2.6 Quincy released"
+date: "2023-04-07"
+author: "yuriw"
+tags:
+  - "release"
+  - "quincy"
+---
+
+This is the sixth backport release in the Quincy series. We recommend all users update to this release.
+
+## Notable Changes
+
+- Cephfs: The 'AT_NO_ATTR_SYNC' macro is deprecated, please use the standard
+  'AT_STATX_DONT_SYNC' macro. The 'AT_NO_ATTR_SYNC' macro will be removed in
+  the future.
+
+- Trimming of PGLog dups is now controlled by the size instead of the version.
+  This fixes the PGLog inflation issue that was happening when the on-line
+  (in OSD) trimming got jammed after a PG split operation. Also, a new off-line
+  mechanism has been added: `ceph-objectstore-tool` got `trim-pg-log-dups` op
+  that targets situations where OSD is unable to boot due to those inflated dups.
+  If that is the case, in OSD logs the "You can be hit by THE DUPS BUG" warning
+  will be visible.
+  Relevant tracker: https://tracker.ceph.com/issues/53729
+
+- RBD: The `rbd device unmap` command gained the `--namespace` option.  Support for
+  namespaces was added to RBD in Nautilus 14.2.0; and while it has been possible to
+  map and unmap images in namespaces using the `image-spec` syntax since then
+  but the corresponding option available in most other commands was missing.
```

Comment:
Changelog looks good, but this "Notable changes" section looks different compared to what you have in https://github.com/ceph/ceph/pull/50721.

Reply:
fixed, good catch, thx @ljflores !